docs/Questions.md

```diff
@@ -0,0 +1,59 @@
+<H1> Folder structure(Pages, Components, Templates, Organisms, Molecules, Atoms) </H1>
+We should stick to one way of organizing our folder and component's structure.
+Utils have files with .tsx file extension, which I think is inappropriate.
+We have validation.js and wdyr.js files at the top level of src
```

Comment:
> We should stick to one way of organizing our folder and component's structure.

Sounds reasonable. Do you feel like we aren't currently doing that? If not, what are the 'differences' where we aren't? What is your proposed structure?

> Utils have files with .tsx file extension, which I think is inappropriate.

I probably agree here. `.tsx` should only be used when JSX/etc is in the file. Do we know why those `utils` files are using `.tsx` currently? Do they have hooks/similar in them?

> We have `validation.js` and `wdyr.js` files at the top level of `src`

Not sure that `validation.js` necessarily has a need to be in the root of `src`, but I assume `wdyr.js` is there because it's imported in `index.tsx`